commands/certmanager/delete.go

```diff
@@ -0,0 +1,69 @@
+package certmanager
+
+import (
+	"context"
+
+	"github.com/ionos-cloud/ionosctl/pkg/config"
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	"github.com/spf13/cobra"
+)
+
+func CertDeleteCmd() *core.Command {
+	cmd := core.NewCommand(context.TODO(), nil, core.CommandBuilder{
+		Namespace: "certmanager",
+		Resource:  "certificates",
+		Verb:      "delete",
+		Aliases:   []string{"g"},
+		ShortDesc: "Delete Certificate by ID",
+		LongDesc:  "Use this command to delete a Certificate by ID.",
+		Example:   "ionsoclt certificate-manager delete --certificate-id 12345678-1234-1234-1234-123456789012",
+		PreCmdRun: PreCmdDelete,
+		CmdRun: CmdDelete,
+		InitClient: true,
+	})
+
+	cmd.AddStringFlag("certificate-id", "", "", "Response get a single certificate (required)")
+	cmd.AddBoolFlag("all", "", false, "Response get a single certificate")
+	_ = cmd.Command.RegisterFlagCompletionFunc(config.ArgCols, func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return allCols, cobra.ShellCompDirectiveNoFileComp
+	})
+
+	return cmd
+}
+
+func CmdDelete(c *core.CommandConfig) error {
+	var err error
+	allFlag := c.Command.Command.Flag("all").Changed
```

Comment:
`.Changed` -- looking at pflag documentation: https://pkg.go.dev/github.com/spf13/pflag#Flag.Changed - `// If the user set the value (or if left to default)` 

would this surely not trigger "delete all" even if we want to delete a single certificate?

Why not use `cmd.Command.Flags().GetBool(constants.ArgAll)`?

Reply:
One more thing to note here, if using `.Changed` - using `ionosctl cert-manager delete --all=false` would probably also trigger `delete all`